heartbeat/pgsql

```diff
@@ -2118,7 +2118,14 @@ check_socket_dir() {
 
 print_crm_mon() {
     if [ -z "$CRM_MON_OUTPUT" ]; then
-        CRM_MON_OUTPUT=`exec_with_retry 0 crm_mon -n1`
+        ocf_version_cmp "$OCF_RESKEY_crm_feature_set" "3.1.0"
+        res=$?
+        if [ -z "$OCF_RESKEY_crm_feature_set" ] || [ $res -eq 2 ]; then
+            XMLOPT="--output-as=xml"
+        else
+            XMLOPT="--as-xml"
+        fi
+        CRM_MON_OUTPUT=`exec_with_retry 0 crm_mon $XMLOPT`
```

Comment:
probably a good idea to use -1, I think it's the default with xml but who knows if that will change one day

Reply:
You probably confuse this with crm_mon parameter?

This is for the exec_with_retry() function, which tells it to retry if it fails.
https://github.com/ClusterLabs/resource-agents/blob/8fea11f81fcd50653a002c87ff26e72622031640/heartbeat/pgsql#L1790

Reply:
Updated to use crm_mon -1 in case the default changes in the future.